lib/api/_loaders/element-global.js

```diff
@@ -156,6 +156,12 @@ class ElementGlobal {
     if (locator instanceof Element) {
       this.locator = locator;
     } else {
+      if (this.client && this.client.locateStrategy){
+        locator = {
+          value: locator,
+          using: this.client.locateStrategy
+        };
+      }
```

Comment:
The case where `use_xpath:true` but we want to use css selector in a command like `element({selector: 'input', using: 'css selector'})` is not handled properly.

Reply:
In this case it should use css selector but it is using xpath

Reply:
fixed in [f6db394](https://github.com/nightwatchjs/nightwatch/pull/3147/commits/f6db394f20385ba472a38bfc82ef60b8b530cb5f)

Reply:
@harshit-bs, I think it would be good if we can add a test for this case as well.